### PR TITLE
Update Place schema

### PR DESCRIPTION
**Pull request by YaleChen299**
#23

### CODE REVIEW

**Comment by YaleChen299** [comment]
seed data for dev will be refactored once i make the test seed factory.

File: tests/review_model_test.py [hunk 1]
```diff
@@ -22,6 +22,7 @@ def test_query_all_reviews(self):
 
     def test_reviews_by_place(self):
         reviews = reviews_by_place('hexcodeplace1')
+        print(reviews)
```
Comment (zhekaiii) [inline tests/review_model_test.py:25]: oops might wanna remove this

File: app/models.py [hunk 2]
```diff
@@ -46,6 +58,13 @@ class Place(db.Model):
     reviews:   so.Mapped[List[Review]] = so.relationship(
         back_populates='place')
     name:      so.Mapped[str] = so.mapped_column(sa.String(64))
+    rating:    so.Mapped[float] = so.mapped_column(sa.Float(asdecimal=True, precision=2), default=0)
+    weighted_rating: so.Mapped[float] = so.mapped_column(sa.Float(asdecimal=True, precision=2), default=0)
+    user_rating_count: so.Mapped[int] = so.mapped_column(sa.Integer, default=0)
+    summary:    so.Mapped[str] = so.mapped_column(sa.String(4096), default='')
+    updated_at: so.Mapped[datetime] = so.mapped_column(TZDateTime)
```
Comment (zhekaiii) [inline app/models.py:65]: Maybe we can put a default here too?
```py
from sqlalchemy import func
```
and then
```suggestion
    updated_at: so.Mapped[datetime] = so.mapped_column(TZDateTime, default=func.now(), onupdate=func.now())
```
Then we don't have to put updated_at anywhere

**Review by zhekaiii — CHANGES_REQUESTED** [review]
Just some minor nits!

**Review by zhekaiii — APPROVED** [review]
LGTM, thanks for the hard work! :)

**Pull request merged**